src/FadeIn.js

```diff
@@ -7,25 +7,28 @@ export default class FadeIn extends Component {
 
   componentDidMount() {
     const count = React.Children.count(this.props.children);
+    var customLag = parseInt(this.props.lag) ? parseInt(this.props.lag) : 50;
```

Comment:
Any reason to `parseInt` here? Might make the API simpler to just accept a number of milliseconds.